src/components/GameList/GameListFilter.svelte

```diff
@@ -88,36 +122,36 @@
 		/>
 	</button>
 	<article id="filter-options" class:filter-options-close={!open} class:filter-options-open={open}>
-		<FilterButton filterTitle="Spelsoort" on:click={() => (spelsoorten = !spelsoorten)} />
+		<FilterButton filterTitle="Spelsoorten" on:click={() => (spelsoorten = !spelsoorten)} />
 		<FilterButton filterTitle="Groepen" on:click={() => (groepen = !groepen)} />
 		<FilterButton
 			filterTitle="Leerlingenaantal"
 			on:click={() => (leerlingenaantal = !leerlingenaantal)}
 		/>
-		<FilterButton filterTitle="Materialen" on:click={() => (materialen = !materialen)} />
+		<!-- <FilterButton filterTitle="Materialen" on:click={() => (materialen = !materialen)} /> -->
 	</article>
 </section>
 
 {#if spelsoorten}
 	<FilterPopUp
-		filterTitle="spelsoort"
+		on:click={testFuntion}
+		filterTitle="category"
 		filterItems={gameNames}
 		on:close={() => (spelsoorten = !spelsoorten)}
 	/>
 {/if}
 
 {#if groepen}
 	<FilterPopUp
-		filterTitle="groepen"
-		{checkedInputs}
+		filterTitle="targetGroup"
 		filterItems={groupNames}
 		on:close={() => (groepen = !groepen)}
 	/>
 {/if}
 
 {#if leerlingenaantal}
 	<FilterPopUp
-		filterTitle="leerlingenaantal"
+		filterTitle="minimumPlayers"
 		filterItems={childrenCount}
 		on:close={() => (leerlingenaantal = !leerlingenaantal)}
 	/>
```

Comment:
Gebruik consistent engels of Nederlands. Dus `filterTitle=targetgroup` dan moet groepen ook `targetGroup` zijn etc